hibernate-reactive-core/src/test/java/org/hibernate/reactive/GeneratedPropertySingleTableTest.java

```diff
@@ -43,13 +42,14 @@
  * Test the @{@link Generated} annotation
  */
 @Timeout(value = 10, timeUnit = MINUTES)
-
+// It requires native queries, so it won't work for every db
+// Fails with SQLSERVER, ORACLE and DB2
+@RunDBType( value = POSTGRESQL )
```

Comment:
In all these cases, you can avoid specifying the attribute.
This will work:
````
@RunDBType( POSTGRESQL )
````

Reply:
done